pkg/plugin/system/bep/bes_pipe.go

```diff
@@ -167,26 +140,29 @@ func (bb *besSocket) publishBesEvent(event *buildeventstream.BuildEvent) error {
 	return eg.Wait()
 }
 
-func (bb *besSocket) Args() []string {
+func (bb *besPipe) Args() []string {
 	return []string{
 		"--build_event_publish_all_actions",
 		// TODO: when bazel6 dropped
 		// "--build_event_binary_file_upload_mode=fully_async",
 		"--build_event_binary_file",
-		bb.socketPath,
+		bb.bepBinPath,
 	}
 }
 
-func (bb *besSocket) RegisterSubscriber(callback CallbackFn) {
+func (bb *besPipe) RegisterSubscriber(callback CallbackFn, multiThreaded bool) {
+	if !multiThreaded {
```

Comment:
Is this ok? I'm not sure which plugins do not support multithreaded...

Reply:
the lint task does _not_ support multi-threaded bes events with the class grpc based backend... as per https://github.com/aspect-build/aspect-cli-legacy/commit/cd6617f8720bd113c00217cda9ee05689e4f7f01. in the bes_pipe, there is no need for multithreaded so we can ignore the value entirely